### ISSUE TITLE
Accessibility: Non-UM Workflow Heading Levels

### ISSUE THREAD

**Issue by jaydonkrooss** [tl-its-umich-edu/canvas-course-manager-next#705]
Problematic component: `MultipleUserEnrollmentWorkflow.tsx`, `CreateSelectSectionWidget.tsx`

Issue summary:
- Non-UM header fix for "Select an existing section to add users to"
- Non-UM header fix for "Add Multiple Users Through CSV"

Based on these reported problems found in [CCM Accessibility Review](https://docs.google.com/spreadsheets/d/1mYrbqwFLLXQoGRTY_XEGkfS3cu3BO_pUN_zQ_DFdhUY/edit?gid=1709548063#gid=1709548063):

- [Severity: Major] UM-Users One Section
    - "Select an existing section to add users to" Should not be "h6" but follow heading sequence or use "p"
- [Severity: Critical] Multiple Non-UM Users
    - "Add Multiple Users Through CSV" Should be h2 not h3
